### DISCUSSION TITLE
Image Optimization problem

### DISCUSSION THREAD

**Discussion by fatdoglabs** [vercel/community#4544] [Bug Report]
### Summary

Previously I was using free vercel service for the my nextjs project(ecommerce site) and suddenly started image optimization was not working because image optmization got full and I updated to the pro plan, and the issue started as auto deployment doesnot work and right now I am using vercel cli to build the project, and then the images issues started where  images of products are not loading on the live site, is there any way to solve this issue. Locally images are loaded properly only issue in live site only.

### Page to Investigate

https://itti-website-nine.vercel.app/

### Steps to Reproduce

I have no idea how to reproduct this issue, anyone can just visit the [site](https://itti-website-nine.vercel.app/), please help me on this as we are making the site live so help will be massive, thankyou

**Comment by amyegan** [comment]
It seems like you may have run into the [Hobby image optimization limit](https://vercel.com/docs/image-optimization/limits-and-pricing#hobby). If that's the case, you would have received email and dashboard alerts when your application got close to the limit.

If you're on a Hobby plan and exceed the limit, new [source images](https://vercel.com/docs/image-optimization/limits-and-pricing#source-images) return a [402 status code](https://vercel.com/docs/errors/platform-error-codes#402:-deployment_disabled) and show the [alt](https://nextjs.org/docs/app/api-reference/components/image#alt) text instead of the image. Images that are already cached continue to work.

**Reply by fatdoglabs** [reply]
Because of the hobby image issue, I have already upgraded to pro plan still images are not showing only alt text is showing

**Reply by fatdoglabs** [reply]
Is there any steps so after new deployment, image optimization might start working

**Reply by amyegan** [reply]
Okay, so the project was on a Hobby plan and got transferred to a Pro plan recently. In that case, the project's repo connection may have a problem. Go to the Git tab in Project Settings, then disconnect and reconnect your repo. That should reset the connection and get automatic deployments working again.

After reconnecting the repo, try pushing a new commit to start a deployment. Does the automatic deployment succeed and do images work in the new deployment?
